test/alternator/test_returnconsumedcapacity.py

```diff
@@ -378,4 +396,22 @@ def test_multi_table_batch_write_item(test_table_s, test_table):
             assert cc["CapacityUnits"] == 5
         else:
             assert cc['TableName'] == test_table.name
-            assert cc["CapacityUnits"] == 2
\ No newline at end of file
+            assert cc["CapacityUnits"] == 2
+
+# batch_write_item has no option to return the previous value
+# This test uses alternator_force_read_before_write to
+# calculate WCU real values
+def test_batch_write_item_with_true_values(test_table, cql):
```

Comment:
Although it became easy to use CQL in Alternator tests when I added that "cql" fixture, I don't think it's a good idea to use it too much.  I would like you to use https://github.com/scylladb/scylladb/pull/19147. Which I'd like to get in, and should be easier than doing stuff in CQL.

Also, it would be a contextmanager, ensuring that the configuration change is undone at the end of the test - even if the test fails. In your current code which doesn't catch exceptions, there are some failure modes that can leave the configuration change.